Program.cs

```diff
@@ -202,6 +203,60 @@ public static void Main(string[] args)
+                    
+                    // mark output as executable
+                    runCommand("chmod", $"+x {stagingPath}/osu.AppDir/");
+
+                    // create AppRun file
+                    string appRunContent = "#!/bin/sh\n" +
+                                           "HERE=\"$(dirname \"$(readlink -f \"${0}\")\")\"\n" +
+                                           "export PATH=\"${HERE}\"/usr/bin/:\"${PATH}\"\n" +
+                                           "EXEC=$(grep -e '^Exec=.*' \"${HERE}\"/*.desktop | head -n 1 | cut -d \"=\" -f 2 | cut -d \" \" -f 1)\n" +
+                                           "exec \"${EXEC}\" $@\n";
+
+                    File.WriteAllText($"{stagingPath}/osu.AppDir/AppRun", appRunContent);
+                    
+                    // make AppRun executable
+                    runCommand("chmod", $"a+x {stagingPath}/osu.AppDir/AppRun");
+                    
+                    // create Desktop file
+                    string desktopFile = "# Desktop Entry Specification: https://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html\n" +
+                                         "[Desktop Entry]\n" +
+                                         "Type=Application\n" +
+                                         "Name=osu!\n" +
+                                         "Comment=Rhythm is just a click away.\n" +
+                                         "Icon=osu!\n" +
+                                         "Exec=osu!\n" +
+                                         "Path=~\n" +
+                                         "Terminal=true\n" +
+                                         "Categories=Game;\n";
+                    
+                    File.WriteAllText($"{stagingPath}/osu.AppDir/osu!.desktop", desktopFile);
+                    
+                    // copy png icon (for desktop file)
+                    File.Copy(Path.Combine(solutionPath, "assets/lazer.png"), $"{stagingPath}/osu.AppDir/osu!.png");
+
+                    // download appimagetool
+                    using (var client = new WebClient())
+                        client.DownloadFile("https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage", $"{stagingPath}/appimagetool.AppImage");
+                    
+                    // mark appimagetool as executable
+                    runCommand("chmod", $"a+x {stagingPath}/appimagetool.AppImage");
+
+                    // create AppImage itself                                       Here are metadata for the AppImage updater, it is based on Github Releases. 
+                    runCommand($"{stagingPath}/appimagetool.AppImage", $"\"{stagingPath}/osu.AppDir\" -u \"gh-releases-zsync|ppy|osu|latest|osu-*x86_64.AppImage.zsync\" \"{Path.Combine(Environment.CurrentDirectory, "releases")}/osu-x86_64.AppImage\"");

```

Comment:
it's quite unclear what the `gh-releases-zsync` portion is doing here. the comments should at very least link to documentation showing what this is doing and how it should be used.